subsys/net/lib/rest_client/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig REST_CLIENT
+	bool "REST client library"
+	select HTTP_CLIENT
+
+if REST_CLIENT
+
+config REST_CLIENT_REQUEST_TIMEOUT
+	int "Rest request timeout, in seconds"
+	default 65
+	help
+	  Sets the timeout duration in seconds to wait for the response data.
+
+config REST_CLIENT_SCKT_SEND_TIMEOUT
+	int "Socket send timeout, in seconds"
+	default 60
+	help
+	  Sets the timeout duration in seconds to use when sending data.
+	  To disable, set the timeout duration to -1.
```

Comment:
Currently setting this to both -1 or 0 disables the timeout. So I think this could be changed so that value 0 is used to disable to timeout instead. And same comment for REST_CLIENT_SCKT_RECV_TIMEOUT.

Reply:
Yes, that could be better as Linux man pages are saying: " If the timeout is set to zero (the default), then the operation will never timeout.". @tokangas, you are sure that Zephyr/libmodem is compliant with that?